libr/bin/format/elf/elf.c

```diff
@@ -2118,8 +2192,26 @@ char* Elf_(r_bin_elf_get_head_flag)(ELFOBJ *bin) {
 		default :                   return strdup (" Unknown mips ISA");
 		}
 	}
-	//TODO: Fill other arch 
-	return strdup("unknown_flag");
+	return NULL;
+}
+
+char* Elf_(r_bin_elf_get_head_flag)(ELFOBJ *bin) {
+	char *head_flag = r_str_new("");
+	char *str = NULL;
+	str = Elf_(r_bin_elf_get_cpu)(bin);
+	if (str) {
+		r_str_append(head_flag, str);
+	}
+	str = Elf_(r_bin_elf_get_features)(bin);
+	if (str) {
+		r_str_append(head_flag, " ");
+		r_str_append(head_flag, str);
+	}
+	
+	if (!*head_flag) {
+		r_str_append(head_flag, "unknown_flag");
```

Comment:
This should be a warning insteaf of appending this wrong feature

Reply:
Not every platform uses ELF head_flag.